java/src/main/java/com/powsybl/dataframe/network/extensions/SlackTerminalDataframeAdder.java

```diff
@@ -0,0 +1,69 @@
+package com.powsybl.dataframe.network.extensions;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.SeriesMetadata;
+import com.powsybl.dataframe.network.adders.AbstractSimpleAdder;
+import com.powsybl.dataframe.update.StringSeries;
+import com.powsybl.dataframe.update.UpdatingDataframe;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.SlackTerminal;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+public class SlackTerminalDataframeAdder extends AbstractSimpleAdder {
+
+    private static final List<SeriesMetadata> METADATA = List.of(
+            SeriesMetadata.stringIndex("voltage_level_id"),
+            SeriesMetadata.strings("element_id"),
+            SeriesMetadata.strings("bus_id"));
+
+    private static class SlackTerminalSeries {
+        private final StringSeries voltageLevelId;
+        private final StringSeries elementId;
+        private final StringSeries busId;
+
+        SlackTerminalSeries(UpdatingDataframe dataframe) {
+            this.voltageLevelId = dataframe.getStrings("voltage_level_id");
+            this.elementId = dataframe.getStrings("element_id");
+            this.busId = dataframe.getStrings("bus_id");
+        }
+
+        void create(Network network, int row) {
+            VoltageLevel voltageLevel = network.getVoltageLevel(this.voltageLevelId.get(row));
+            if (voltageLevel == null) {
+                throw new PowsyblException("voltage_level_id : " + this.voltageLevelId.get(row) +
+                        " does not correspond to any voltage level");
+            }
+            if (this.elementId != null && this.busId == null) {
+                Identifiable identifiable = network.getIdentifiable(this.elementId.get(row));
+                Terminal terminal = ((Injection) identifiable).getTerminal();
```

Comment:
We should check the type of the identifiable (almost as it was done before):
- if it's an injection, do this
- if it's a bus, it means that it's a configured bus of a bus-breaker voltage level --> in this case we attach the slack to it.
  It's different from the "bus_id" column, because in bus_id we expect only buses from the bus view
- else, throw an exception saying that we only handle injections, busbars, or configured buses as an "element_id"